.github/workflows/ci.yml

```
@@ -31,10 +31,14 @@ jobs:
       - name: Install numpy
         run: python3 -m pip install numpy
 
-      - name: Install OpenMP
+      - name: Install OpenMP on linux
         if: matrix.os == 'ubuntu-latest'
         run: sudo apt-get install -y libomp-dev
 
+      - name: Install OpenMP on macOS
```

Since below we already used `macos` at the line of "Add .bazelrc.user on macos", please use it

```suggestion
      - name: Install OpenMP on macos
```